alarms/utils.py

```diff
@@ -278,6 +278,13 @@ def get_timestamps(t):
 	time_24 = disappear_time.strftime("%H:%M:%S")
 	return (time_left, time_12, time_24)
 	
+# Check if the Pokemon will be back after a hidden phase
+# Addinfo values: 1=2x15 point, 2=1x60h2 point, 3=1x60h3 point, 4=1x60h23 point, only needs to be added, if scanned before the break
+def get_addinfo(addinfo):
```

Comment:
Same here - rename to get_spawn_text or something similiar - otherwise it might be mistaken as 'address info' or something.